spaces/viralata.eth/my-space.scss

```
@@ -0,0 +1,10 @@
+.my-space {
```

The css selector should similar to the skin file name. `skin.scss` `.skin {}`.
```suggestion
.viralata {
```
Your skin should not be added in your space directory but should be added in this directory https://github.com/snapshot-labs/snapshot-spaces/tree/master/skins
Change `spaces/viralata.eth/my-space.scss` to `skins/viralata.scss`